modules/identity/.editorconfig

```diff
@@ -0,0 +1,131 @@
+# Rules in this file were initially inferred by Visual Studio IntelliCode from the D:\Projects\Volosoft\abp\framework codebase based on best match to current usage at 2.10.2020.
```

Comment:
We follow the same rules for all the solutions in the abp & volo repositories. So, we don't like to add `.editorconfig` to every individual solution. It would not be maintainable.
If we add such a file, we should add in a single place per GitHub repository.

Reply:
I don't know how to best handle this. I have too many projects and solutions that are using my own formating and only ABP projects have a default format. That's why I added these `.editorconfig` files. 

Copying and removing `.editorconfig` before and after each push/pull is very practical...

Reply:
I think you add this file to .gitignore because no one needs it except you :)

Reply:
Yep, that's a good idea :) tnx

Reply:
Removed files and added new rule to gitignore